### PR TITLE
Support image based class weighted CE

### PR DESCRIPTION
**Pull request by Junjun2016**
Support image based class weighted CE

### CODE REVIEW

File: mmseg/models/losses/cross_entropy_loss.py [hunk 1]
```diff
@@ -6,22 +6,58 @@
...
+    else:
+        weights = ((hist != 0) * upper_bound * (1 - hist_norm)) + 1
+    return weights
+
+
 def cross_entropy(pred,
                   label,
                   weight=None,
+                  img_based_class_weights=None,
+                  batch_weights=True,
```
Comment (xvjiarui) [inline mmseg/models/losses/cross_entropy_loss.py:24]: I suggest making `CrossEntropy` simple as it is. 
We may add a new loss module.
